generator/src/components/Button/index.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  export let onClick: () => void
```

Comment:
The same comment to this applies from the other PR, why would we want a button without an `onclick` handler

Reply:
onClick is always required!

Reply:
oh indeed, I misread this